templates/index.html

```diff
@@ -325,6 +328,43 @@ <h3 class="p-heading--2">Simple, secure and scalable IoT</h3>
         </div>
       </div>
     </div>
+    <div tabindex="0" role="tabpanel" id="Automotive-tab" aria-labelledby="Automotive" aria-hidden="false">
+      <div class="p-block">
+        <div class="row">
+          <div class="col-2 col-medium-2 col-small-2">
+            {{ image (
+            url="https://assets.ubuntu.com/v1/0857c6ff-Microvision_logo_white.svg",
+            alt="Microvision logo",
+            width="201",
+            height="138",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+            }}
+          </div>
+          <div class="col-9 col-medium-4 col-start-large-4 col-small-4">
+            <div class="row p-block">
+              <div class="col-6">
+                <h3 class="p-heading--2">Private clouds that speed up innovation</h3>
+              </div>
+              <div class="col-3">
+                <p>MicroVision accelerates their development team’s VM deployments with an OpenStack-based private cloud, ideal for innovation in autonomous driving. </p>
+                <p><a href="https://ubuntu.com/engage/microvision-powers-autonomous-driving-with-charmed-openstack">Read the case study&nbsp;&rsaquo;</a></p>
+              </div>
+            </div>
+            {{ image (
+            url="https://assets.ubuntu.com/v1/fd9cffd1-Test_Vehicle_Hamburg3.JPG",
+            alt="Vehicle image",
```

Comment:
```suggestion
            alt="",
```
Decorative images usually shouldn't have alt text, more [here](https://www.w3.org/WAI/tutorials/images/decorative/)